client/app/app.module.ts

```diff
@@ -4,17 +4,15 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
 import { RouterModule } from '@angular/router';
 
+import { routes } from './config/routes';
 import { AppComponent } from './app.component';
 import { HomeComponent } from './home/home.component';
 import { AboutComponent } from './about/about.component';
 import { DataService } from './services/data.service';
 
 import { ToastComponent } from './shared/toast/toast.component';
 
-const routing = RouterModule.forRoot([
-    { path: '',      component: HomeComponent },
-    { path: 'about', component: AboutComponent }
-]);
+const routing = RouterModule.forRoot(routes,  { useHash: true });
```

Comment:
Ok, since there is no PageNotFound component now, I'd go back to normal HTML5 routing, that I prefer.